tests/integ/modin/frame/test_fillna.py

```diff
@@ -150,6 +150,23 @@ def test_value_scalar(test_fillna_df):
     )
 
 
+@sql_count_checker(query_count=2)
+def test_timedelta_value_scalar(test_fillna_df):
+    timedelta_df = test_fillna_df.astype("timedelta64[ns]")
+    eval_snowpark_pandas_result(
+        pd.DataFrame(timedelta_df),
+        timedelta_df,
+        lambda df: df.fillna(pd.Timedelta(1)),  # dtype keeps to be timedelta64[ns]
+    )
+
+    # Snowpark pandas dtype will be changed to int in this case
+    eval_snowpark_pandas_result(
+        pd.DataFrame(timedelta_df),
+        test_fillna_df,
+        lambda df: df.fillna(1),
+    )
```

Comment:
```suggestion
@sql_count_checker(query_count=1)
@pytest.mark.parametrize("value", [
	pd.Timedelta(1), # dtype keeps to be timedelta64[ns]
	1 # Snowpark pandas dtype will be changed to int in this case
)
def test_timedelta_value_scalar(test_fillna_df, value):
    timedelta_df = test_fillna_df.astype("timedelta64[ns]")
    eval_snowpark_pandas_result(
        pd.DataFrame(timedelta_df),
        timedelta_df,
        lambda df: df.fillna(value)
     )
```

Reply:
We cannot do this because when value is 1, the column becomes int in snowpark but is a variant in pandas.